shell/ci/release/docker.sh

```diff
@@ -139,24 +159,26 @@ build_and_push_image() {
       buildContext="$(get_repo_directory)/deployments/$image"
     fi
   fi
+  args+=("$buildContext")
+
+  # Check if the arguments include --push, if so, we'll push
+  # the image after building it.
+  push=false
+  for arg in "${args[@]}"; do
+    if [[ $arg == "--push" ]]; then
+      push=true
+    fi
+  done
 
-  if [[ -n $CIRCLE_TAG ]]; then
-    echo "🔨 Building and Pushing Docker Image (production)"
-    (
-      set -x
-      docker buildx build "${args[@]}" --platform "$platformArgumentString" \
-        -t "$remote_image_name:$VERSION" -t "$remote_image_name:latest" --push \
-        "$buildContext"
-    )
+  if [[ $push == true ]]; then
+    echo "🔨 Building and Pushing Docker Image"
   else
-    # When we're not on a tag, just build the docker image to verify
-    # that it is buildable.
-    info "Building Test Docker Image"
-    (
-      set -x
-      docker buildx build "${args[@]}" -t "$image" --load "$buildContext"
```

Comment:
Note: now that we're not scanning the image, we no longer need `--load`. So, I removed it. This should be marginally faster. But also, we can't load multi-arch docker images currently.